vue/components/ui/molecules/scrollable-items/scrollable-items.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+    <div class="scrollable-items">
+        <button-icon
+            class="button-icon-previous"
+            v-bind:icon="'chevron-left'"
+            v-bind:disabled="isButtonPreviousDisabled"
+            v-bind="_buttonIconProps"
+            v-if="showArrows"
+            ref="button-icon-previous"
+            v-on:click="onButtonIconPreviousClick"

```

Comment:
Try to simplify a bit. This applies to other stuff like `isButtonPreviousDisabled `

```suggestion
            v-on:click="onPreviousClick"

```